backend/src/case_file/case_file.service.ts

```diff
@@ -1390,7 +1521,9 @@ export class CaseFileService {
   // find all equipment records, and their respective actions, for a given case
   // Since we want to list the equipment related to a case, rather than the actions for a case, which may contain equipment, let's
   // transform the actions with equipment to equipment with actions.
-  private findEquipmentDetails = async (caseIdentifier: string): Promise<Equipment[]> => {
+  private findEquipmentDetails = async (
```

Comment:
@barrfalk This is working as expected, but there are a couple things to point out for future: 

```
//-- current
for (const action of actions) {
      const equipment = action.equipment; 
}
```

```
//-- suggested
//-- using deconstructed vars makes things easier to read as you're not trying to follow the dot notation.
//-- not a huge deal on smaller objects but can make a big difference for larger objects
for (const action of actions) {
      const { equipment } = action
      const {
         active_ind, equipment_guid, create_utc_timestamp, equipment_code, equipment_location_desc ,
      } = equipment;
}
```

```
//-- red flag
//-- this works and there's nothing technically wrong here, but you're muting the actions array when you're 
//-- iterating over the actions. It would be better to instead of just mutating the array, recreate it to avoid 
//-- a potential side effect
        equipmentDetail.actions.push({
          actionGuid: action.action_guid,
          actor: action.actor_guid,
          date: action.action_date,
          activeIndicator: action.active_ind,
          actionCode: actionData.action_code,
        });
```
```
//-- suggestion
 equipmentDetail.actions = [...equipmentDetail.actions, {
          actionGuid: action.action_guid,
          actor: action.actor_guid,
          date: action.action_date,
          activeIndicator: action.active_ind,
          actionCode: actionData.action_code,
        }]
```